handlers/oam_handler.go

```diff
@@ -76,7 +102,7 @@ func getAppropriateCueVal(value string, valType string) (cue.Value, error) {
 // 	200:
 
 // request body should be json
-// request body should be of format - {schema: string, schemaType: "JSON" | "YAML" | "JSONSCHEMA", value: string, valueType: "JSON" | "YAML" }
+// request body should be of format - {schema: string, schemaType: "JSON" | "YAML" | "JSONSCHEMA", value: string, valueType: "JSON" | "YAML" } or {list: []validationInfo}
 // it will respond with error if any occurred, or with `isValid: bool` json
 func (h *Handler) ValidationHandler(rw http.ResponseWriter, r *http.Request) {
```

Comment:
This handler is part of coming `MeshModel` and not used currently anywhere right?

Reply:
For the most part, yes. It is also being used in MeshMap [here](https://github.com/layer5labs/meshery-extensions/pull/1006/files#diff-6c003e5b640a294ef99c2c9e2509e29c38d9671784fe39373905bfd24c6adc08R8)